service/src/test/java/bio/terra/pipelines/testutils/MockMvcUtils.java

```diff
@@ -50,7 +50,7 @@ public static String convertToJsonString(Object obj) throws JsonProcessingExcept
   public static final String TEST_WORKSPACE_NAME = "testTerraWorkspaceName";
   public static final String TEST_WORKSPACE_STORAGE_CONTAINER_NAME = "test-bucket-name";
   public static final String TEST_WORKSPACE_GOOGLE_PROJECT = "testGoogleProject";
-  public static final String TEST_WDL_METHOD_VERSION = "0.12.1";
+  public static final String TEST_WDL_METHOD_VERSION = "0.1.12";
```

Comment:
you didnt like my fake version???

Reply:
lol it didn't match the others and there's some cross-talk in the tests

Reply:
but oops that was actually a signal that something was wrong. reverted and fixed a bug! thanks!!